Resources/Prototypes/Catalog/Fills/Crates/fun.yml

```diff
@@ -32,7 +32,8 @@
         amount: 1
       - id: PlushieAtmosian
         amount: 1
-
+      - id: PlushieSoyjak
+        amount: 1
```

Comment:
This should be added to some plushie spawners and/or mail, if it's meant to show up in more places than just the plushie crate orderable from cargo.